style.css

```diff
@@ -0,0 +1,519 @@
+  margin-right: 7%;
+  width: 86.6%;
+}
+
+.gridd {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.my-recent-works {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  background-color: white;
+  flex: 1;
+  height: 100vh;
+  width: 100vw;
+  padding-bottom: 10%;
+}
+
+.H1 {
+  position: relative;
+  width: 100%;
+  margin-top: 15%;
+  margin-left: 10%;
+  font-size: 24px;
+  align-self: center;
+  color: #1c1a19;
+}
+
+.line {
+  height: auto;
+  display: flex;
+  position: relative;
+  margin-left: 2.5%;
+  width: 100%;
+  margin-top: 2.5%;
+}
+
+.main-image {
+  display: flex;
+  margin-top: 15%;
+  width: 100%;
+  text-align: center;
+}
+
+.H2 {
+  margin-left: 0;
+  position: relative;
+  width: 90%;
+  margin-top: 5%;
+  font-size: 18px;
+  color: #1c1a19;
+  order: 0;
+}
+
+.p-recent > p {
+  position: relative;
+  width: 80%;
+  margin-left: 0;
+  margin-right: 10%;
+  flex: 1;
+  overflow: hidden;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.buttons-0 {
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  list-style-type: none;
+  margin-top: 5%;
+  margin-left: 0%;
+  margin-right: 10%;
+  width: 100%;
+  flex-wrap: wrap;
+}
+
+.buttons-0 > li > button {
+  border: 1px solid #8993a4;
+  padding: 10px;
+  background: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.css0 {
+  margin-left: 10%;
+}
+
+.html0 {
+  margin-left: 20%;
+}
+
+.bootstrap0 {
+  margin-left: 30%;
+}
+
+.ruby0 {
+  margin-left: 70%;
+}
+
+.see-project {
+  display: flex;
+  flex: 1;
+  width: 130px;
+  height: 48px;
+  position: relative;
+  margin-top: 20px;
+  background-color: #ff6b00;
+  margin-left: 0;
+  align-items: center;
+}
+
+.see-project > a {
+  margin-left: 20%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: center;
+}
+
+.see-project > a :hover {
+  border: 2px solid blue;
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.btn:hover {
+  border: 2px solid blue;
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+/* profesional-art */
+
+.profesional-art-1 {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  justify-content: center;
+  position: relative;
+  width: 90vw;
+  height: 50vh;
+  background-image: url(image/background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.H3 {
+  position: relative;
+  margin-top: 30%;
+  width: 90%;
+  height: 81px;
+  left: 10px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 28px;
+  line-height: 44px;
+  color: #fff;
+  overflow: hidden;
+}
```

Comment:
Hello, @solog0039 as requested by the previous reviewer, the text here gets embedded into the card depending on the screen size. I see you provided a solution to it but it is only on certain screen sizes. I opened your project across multiple screens and found this issue. Let's add a little bit more CSS here to fix this problem so that your project looks great on different mobile screens.
The following are screenshots of your app but on different screen sizes.

375px
![image](https://user-images.githubusercontent.com/29071984/170774948-4212b9f7-7a73-483f-8193-f1310d973d42.png)

414px
![image](https://user-images.githubusercontent.com/29071984/170775186-4ceef47a-9cd4-43b3-91d0-bfae9670db95.png)